modules/stringfier.js

```diff
@@ -0,0 +1,6 @@
+import Store from './store.js';
+
+export default function Stringifier() {
```

Comment:
Kindly use es6 arrow function syntax. This is good practice and is also a requirement for this milestone
<img width="1673" alt="Screenshot 2022-11-01 at 14 35 03" src="https://user-images.githubusercontent.com/48546992/199225433-63445163-680e-4757-aa2f-d69080a5ad01.png">